src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -11,33 +11,33 @@ namespace Microsoft.ML
     public static class TextLoaderSaverCatalog
     {
         /// <summary>
-        /// Create a text reader <see cref="TextLoader"/>.
+        /// Create a text loader <see cref="TextLoader"/>.
         /// </summary>
         /// <param name="catalog">The <see cref="DataOperations"/> catalog.</param>
         /// <param name="columns">The columns of the schema.</param>
         /// <param name="hasHeader">Whether the file has a header.</param>
         /// <param name="separatorChar">The character used as separator between data points in a row. By default the tab character is used as separator.</param>
         /// <param name="dataSample">The optional location of a data sample.</param>
```

Comment:
>/// <param name="dataSample">The optional location of a data sample.</para [](start = 8, length = 74)

?Do you want to also create a small sample, for the apis of this file, and link it from here?